.github/workflows/end2end-test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             - name: Npm install and build
               run: |
                   npm ci
-                  FORCE_REDUCED_MOTION=true npm run build
```

Comment:
should we remove the usage of `FORCE_REDUCED_MOTION` in the code.

Reply:
I'm not sure if it's used by anyone or anything else. But sure, I think we should remove it.